great_expectations/core/util.py

```diff
@@ -406,25 +406,51 @@ def datetime_to_int(dt: datetime.date) -> int:
 
 class AzureUrl:
     """
-    Parses an Azure Blob Storage URL into its separate components
-    Format: <ACCOUNT_NAME>.blob.core.windows.net/<CONTAINER>/<BLOB>
+    Parses an Azure Blob Storage URL into its separate components.
+    Formats:
+        WASBS (for Spark): "wasbs://<container name>@<account name>.blob.core.windows.net/<directory name>/<file name>"
+        HTTP(S) (for Pandas) "<ACCOUNT_NAME>.blob.core.windows.net/<CONTAINER>/<BLOB>"
```

Comment:
Two comments:
1. A blob is the absolute path of a file in a container (directories included). I think instead of making a distinction between directory and file, we should just put blob (this is represented in the below regex).
2. Could we keep the format of these placeholders consistent?

i.e. `wasbs://<CONTAINER>@<ACCOUNT_NAME>.blob.core.windows.net/<BLOB>`

Reply:
@cdkini Updated -- thanks!